server/data/prisonApiClient.ts

```diff
@@ -286,6 +290,24 @@ export default class PrisonApiRestClient implements PrisonApiClient {
     return this.get<StaffRole[]>({ path: `/api/staff/${staffId}/${agencyId}/roles` })
   }
 
+  async getAgencyDetails(agencyId: string): Promise<Agencies> {
```

Comment:
Would it make sense to have `activeOnly` as a method argument, so other callers can set it to true if needed?